.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/*
```

Comment:
You shoul rereade the task.
You did the opposite.
You should delete logs/* and push you commit with logs file.

Note that you should push the file with logs to your PR, so please, do not add it to .gitignore. You may probably need to use the absolute path to the log file in log4j2.xml.